assayist/processor/container_rpm_analyzer.py

```diff
@@ -109,7 +123,8 @@ def _get_rpms_diff(self, parent_archive_id, child_archive_id):
 
         child_rpm_ids = set()
         id_to_rpm = {}
-        for rpm in self.koji_session.listRPMs(imageID=child_archive_id):
+        image_rpm_file = self.read_metadata_file(self.IMAGE_RPM_FILE)
+        for rpm in image_rpm_file.get(str(child_archive_id)):
```

Comment:
Could you make the `.get` command default to `[]` as in `image_rpm_file.get(str(child_archive_id), [])`, that way the for loop won't crash if it encounters `None`. We should probably also log in this case and just return an empty list.

Reply:
Added default empty list and a log statement in case the list ends up being empty.